### PR TITLE
Bmt 2024

### PR DESCRIPTION
**Pull request by oliver-ni**

### CODE REVIEW

File: content/events/bmt-2024.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+#### General Test (90 mins)
+
+The General Test is an individual round consisting of questions across the
+various subjects that comprise the focus tests. This round is highly recommended
+for students with less prior contest experience.
+
+#### Focus Tests (60 mins)
+
+There are four focus tests of 10 questions across four subjects --- Algebra,
```
Comment (lethalNeutrino) [inline content/events/bmt-2024.md:107]: This is not how to make an emdash in markdown — you want to put the actual character there

File: content/events/bmt-2024.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+
+When weighting team scores, scores for a given test will be normalized by
+either the top 10 scores or the top 10% score, whichever is greater.
+Additionally, if a student takes the General Test instead of the two Focus
+Tests, their score will only count for half as many points as the Focus
+combination would.
+
+## Tiebreakers
+
+This year, to streamline the event, we have a new automatic tiebreaker policy,
```
Comment (lethalNeutrino) [inline content/events/bmt-2024.md:135]: Commas are weird here, maybe "To streamline the event this year,"

File: content/events/bmt-2024.md [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+Graph paper and calculators are prohibited, but protractors, rulers, and
+compasses are permitted. Blank white scratch paper will be provided for all
+participants. All answers must be exact, reduced, and simplified. Illegible
+answers will not be graded. Cheating in any form will not be tolerated, and
+failure to comply with all rules may result in immediate and irreversible
+disqualification.
+
+### Can middle school students compete? What if I don’t have a team?
+
+Yes! BMT is open to any student in 8th grade or below, including those who
```
Comment (lethalNeutrino) [inline content/events/bmt-2024.md:296]: BMT is open to students in 12th grade or below, not 8th grader or below

File: content/events/bmt-2024.md [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+Awards will be physically presented to contestants at the closing ceremony.
+
+### I’m in a different timezone or non-local and can’t compete live. What are my options?
+
+There will be BMT Satellite competitions held through our partners across the
+globe for competitors wishing to compete in a different time zone.
+
+### Will lunch be provided?
+
+Lunch is **not provided**. Students, coaches and parents are free to find food
```
Comment (lethalNeutrino) [inline content/events/bmt-2024.md:317]: we should make a decision on this lol also should we change the word homerooms

File: content/news/2024-09-11_bmt-2024-registration-opens-soon.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
++++
+title = "Registration for BMT 2024 will open on September 16, 2024"
++++
+
+How are you doing, fellow math enthusiasts?
+
+BMT is coming back on Saturday, November 2nd, and we're excited to see you
+there! Registration will open on **Monday, September 16 at 12:01 AM PT**. This
```
Comment (lethalNeutrino) [inline content/news/2024-09-11_bmt-2024-registration-opens-soon.md:8]: change open time — see discord

File: templates/index.html [hunk 6]
```diff
@@ -79,39 +79,39 @@ <h2 class="font-bold text-xl md:text-3xl">Recent & Upcoming Events</h2>
...
   name="BMT China 🇨🇳",
-  date="December 2–3, 2023",
+  date="Nov 30 – Dec 1, 2024",
   href="http://www.seedasdan.asia/en/bmt-en/",
   description="Our partner ASDAN serves students in Mainland China, Taiwan, Hong Kong, Macau, and nearby regions."
   ) }}
 
   {{ self::event(
   class="col-span-6 bg-gradient-to-br from-slate-100 to-slate-300",
   name="BMT 2023 Online",
```
Comment (lethalNeutrino) [inline templates/index.html:100]: fix name to BMT Online 2024

File: templates/index.html [hunk 7]
```diff
@@ -79,39 +79,39 @@ <h2 class="font-bold text-xl md:text-3xl">Recent & Upcoming Events</h2>
...
+  name="Berkeley Math Tournament 2024",
+  date="November 2, 2024",
+  href="/events/bmt-2024/",
   description="Our live fall tournament offers teams of high school students the opportunity to share their interest in
   mathematics with like-minded individuals and to get a taste of the concepts studied in higher mathematics."
   ) }}
 
   {{ self::event(
   class="col-span-2 bg-gradient-to-br from-slate-100 to-slate-300",
   name="BMT China 🇨🇳",
```
Comment (lethalNeutrino) [inline templates/index.html:92]: Possibly add a 2024?

**Review by lethalNeutrino — CHANGES_REQUESTED** [review]

File: content/events/bmt-2024.md (hunk 3 above)
Comment (oliver-ni) [reply content/events/bmt-2024.md:296]: wtf LOL

File: content/events/bmt-2024.md [hunk 8]
```diff
@@ -124,40 +125,45 @@ Team scores are determined based on a combination of teams' scores on the Guts
...
-students correctly answered precisely the same set of problems. In this case,
-those students will be called in for an additional tiebreaker.
+If a tie remains on the Individual Round after the preceding automatic
+tiebreaking procedure -- i.e., if two students correctly answered precisely the
+same set of problems, then those students will participate in an additional
+Tiebreaker Round. Ties that remain on the Guts Round or Power Round will be
+tiebroken.
 
-This 15-minute test will consist of three questions, and be scored primarily on
+This 15-minute test will consist of five questions, and be scored primarily on
```
Comment (annoyingpi) [inline content/events/bmt-2024.md:166]: three questions for Focus Tests; five questions for the General Test

**Pull request merged**